templates/agents/full-stack-developer.md

```
@@ -0,0 +1,118 @@
+/src
+  /app                  # Next.js App Router pages and API routes
+  /components/ui        # shadcn/ui components
+  /components           # Domain-specific components
+  /lib                  # Utilities, Supabase clients
+  /hooks                # Custom React hooks
+  /types                # TypeScript types (including database.types.ts)
+```
+
+## Universal Response Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)